external/docs/content/docs/gitformat-signature.html

```diff
@@ -46,7 +46,7 @@ <h2 id="_description"><a class="anchor" href="#_description"></a>DESCRIPTION</h2
 </div>
 <div class="dlist">
 <dl>
-<dt class="hdlist1" id="Documentation/gitformat-signature.txt-codegpgcodePGP"> <a class="anchor" href="#Documentation/gitformat-signature.txt-codegpgcodePGP"></a><code>gpg</code> (PGP) </dt>
+<dt class="hdlist1" id="Documentation/gitformat-signature.txt-codegpgcodePGP"> <a class="anchor" href="#Documentation/gitformat-signature.txt-codegpgcodePGP"></a><code>opengpg</code> (PGP) </dt>
```

Comment:
Please note the notice at the top of that file: https://github.com/git/git-scm.com/blob/8e2ef7c8fed8749544df8ebecb6e3877080baba4/external/docs/content/docs/gitformat-signature.html#L2

The original source of this is actually here: https://github.com/git/git/blob/v2.47.1/Documentation/gitformat-signature.txt#L28.

Please follow the guidelines at https://git-scm.com/community's Contributing to Git to propose the patch on the Git mailing list instead.